src/urllib3/exceptions.py

```diff
@@ -48,16 +62,17 @@ class SSLError(HTTPError):
 class ProxyError(HTTPError):
     """Raised when the connection to a proxy fails."""
 
-    def __init__(self, message, error, *args):
-        super().__init__(message, error, *args)
+    original_error: Exception
+
+    def __init__(self, message: str, error: Exception) -> None:
+        super().__init__(message, error)
         self.original_error = error
 
 
 class HTTPSProxyError(ProxyError):
     """Used only when establishing a TLS connection to a proxy"""
 
-    def __init__(self, message, error, *args):
-        super().__init__(message, error, *args)
+    pass
```

Comment:
Removed the `__init__` that was added here; it's technically written incorrectly, since `super().__init__()` can't take `*args`, and removing that item results in an identical signature to its parent, so we just inherit that instead.